data-serving/data-service/src/controllers/case.ts

```diff
@@ -18,6 +21,35 @@ export const get = async (req: Request, res: Response): Promise<void> => {
     res.json(c);
 };
 
+/**
+ * Streams a CSV attachment of all cases.
+ *
+ * Handles HTTP GET /api/cases/download.
+ */
+export const download = async (req: Request, res: Response): Promise<void> => {
+    const cases = await Case.find({}).lean();
+
+    res.setHeader('Content-Type', 'text/csv');
+    res.setHeader('Content-Disposition', 'attachment; filename="cases.csv"');
+    res.setHeader('Cache-Control', 'no-cache');
+    res.setHeader('Pragma', 'no-cache');
+    fetch(
```

Comment:
let's use axios for http requests as we do in the curator service, unless there's a real reason to use this new library.

Reply:
Done